msgpack-core/src/test/scala/org/msgpack/core/MessageUnpackerTest.scala

```diff
@@ -451,7 +470,7 @@ class MessageUnpackerTest extends MessagePackSpec {
         }
 
         block("v7") {
-          val unpacker = MessagePack.newDefaultUnpacker(data)
+          val unpacker = Random.shuffle(unpackers(data)).head
```

Comment:
@xerial I've made changes based on the comments, https://github.com/msgpack/msgpack-java/pull/380 on the directbuf branch